### DISCUSSION TITLE
v0.13.4

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1540] [Releases]
**Attention**: Now requires libsignal-client version 0.47.0

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.13.3...v0.13.4

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.13.4'>v0.13.4</a>.</em>

**Comment by indrasensri** [comment]
Any one can install for me
